backend/api/v1/project_service.go

```diff
@@ -266,6 +266,128 @@ func (s *ProjectService) SetIamPolicy(ctx context.Context, request *v1pb.SetIamP
 	return convertToIamPolicy(iamPolicy), nil
 }
 
+// GetDeploymentConfig returns the deployment config for a project.
+func (s *ProjectService) GetDeploymentConfig(ctx context.Context, request *v1pb.GetDeploymentConfigRequest) (*v1pb.DeploymentConfig, error) {
+	trimDeploymentConfigSuffix, err := trimSuffix(request.Name, deploymentConfigSuffix)
```

Comment:
Consider moving this to a separate function in common.go, since this boring and lengthy stuff has occurred twice. The function checks all errors and returns a project.